tests/phpunit/tests/Checker/Checks/Plugin_Readme_Check_Tests.php

```diff
@@ -95,6 +95,26 @@ public function test_run_with_errors_license() {
 		$this->assertEquals( 'invalid_license', $warnings['readme.txt'][0][0][0]['code'] );
 	}
 
+	public function test_run_with_errors_no_license() {
+		$readme_check  = new Plugin_Readme_Check();
+		$check_context = new Check_Context( UNIT_TESTS_PLUGIN_DIR . 'test-plugin-plugin-readme-errors-no-license/load.php' );
+		$check_result  = new Check_Result( $check_context );
+
+		$readme_check->run( $check_result );
+
+		$errors = $check_result->get_errors();
```

Comment:
```suggestion
		$errors   = $check_result->get_errors();
		$warnings = $check_result->get_warnings();

		$this->assertEmpty( $warnings );
		$this->assertSame( 0, $check_result->get_warning_count() );
```

For safe guard check empty warning.